policy/lib/bundles.rego

```diff
@@ -57,6 +57,13 @@ unacceptable_task_bundle(tasks) := {task |
 	_is_unacceptable(ref)
 }
 
+# Returns if the task uses an acceptable bundle reference
+is_acceptable(task) if {
```

Comment:
NIT: Could we rename this to `is_acceptable_task` to make it clear that this is not an alternative to `not _is_unacceptable`?  Also, I wonder if this could be implemented like this instead:

```
is_acceptable_task(task) if {
  ref := image.parse(_bundle_ref(task, _task_bundles))
  not _is_unacceptable(ref)
}
```

`_is_unacceptable` takes into account `_newer_in_effect_version_exists` which I think should also be done here.

Reply:
I opted not to check if the task is latest as `task_ref_bundles_current` will check that already, checking that here again would cause two rule violations for out of date task bundles.

Reply:
Valid point, let's examine this in https://github.com/enterprise-contract/ec-policies/issues/858